.env.example

```diff
@@ -2,3 +2,4 @@ POSTGRES_PASSWORD=password
 DATABASE_URL=postgres://postgres:$POSTGRES_PASSWORD@localhost:5432
 GOVUK_NOTIFY_API_KEY=
 SIGN_IN_METHOD=persona
+GIAS_DOWNLOAD_LINK=https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public
```

Comment:
Is this URL correct? It returns a 404.

```
$ curl -I https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public
HTTP/2 404
```

Reply:
Well it works only if you put the correct file in there,  Something like `https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public/edubasealldata20231213.csv`

Which is dependent on `Day.today`, so there might not be too much value putting it in here as it doesn't really function on it's own it needs more [ruby logic ](https://github.com/DFE-Digital/itt-mentor-services/blob/cced3e2b6aece6c68fc518900181f9105f83269a/lib/tasks/gias_update.rake#L10)